index.html

```diff
@@ -6,216 +6,266 @@
             </div>
-
+            <h2 class="holder-title">Frameworks</h2>
+            <ul class="card-tags-container">
+                <li class="tags-item">Tailwind</li>
+                <li class="tags-item">Bootstrap</li>
+                <li class="tags-item">Selenium</li>
+                <li class="tags-item">RSpec</li>
+                <li class="tags-item">GraphQL</li>
+            </ul>
         </div>
 
         <div class="card-holder-container">
             <div class="holder-image">
-                <img src="./img/icon-skills.svg" alt="icon skills">
-            </div>
-            <h2 class="holder-title">
-                Skills
-            </h2>
-            <div class="card-tags-container">
-                <div class="tags-item">Codekit</div>
-                <div class="tags-item">GitHub</div>
-                <div class="tags-item">CodePen</div>
-                <div class="tags-item">GitLab</div>
-                <div class="tags-item">Terminal</div>
+                <img src="./img/icon-skills.svg" alt="icon skills" />
             </div>
+            <h2 class="holder-title">Skills</h2>
+            <ul class="card-tags-container">
+                <li class="tags-item">Codekit</li>
+                <li class="tags-item">GitHub</li>
+                <li class="tags-item">CodePen</li>
+                <li class="tags-item">GitLab</li>
+                <li class="tags-item">Terminal</li>
+            </ul>
         </div>
-
     </section>
 
-    <h1 class="last-title-container">
-        I'm always interested in hearing about new projects, so if you'd like to chat please get<br> in touch.
-    </h1>
+    
+<footer class="footer-container" id="footer-container-id">
+    <div class="footer-form">
+        <h1 class="footer-title">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
+        <form action="https://formspree.io/f/moqbyree" method="post" class="form-input">
+            <ul class="form-fields">
+                <li class="input-name">
+                    <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
+
+                </li>
+                <li class="input-full-name">
+                    <input type="text" name="first_name" id="first_name" maxlength="30" placeholder="First name"
+                        required>
+                    <input type="text" name="last_name" id="last_name" maxlength="30" placeholder="Last name"
+                        required>
+                </li>
+                <li class="input-email-li">
+                    <input class="input-email" type="email" name="email" id="email" placeholder="Email address"
+                        required>
+                </li>
+                <li>
+                    <textarea name="text_area" id="text_area" cols="30" rows="10" maxlength="500"
+                        placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] Please make sure that your mobile version stay the same. but I think there is some thing that to be improved in your mobile version design. kindly add the background image into your mobile version.
### your design👇
![contst](https://user-images.githubusercontent.com/66992901/201481104-b7761a1c-84ca-4051-9e1b-2e4314304307.PNG)
### figma template👇
![contfigma](https://user-images.githubusercontent.com/66992901/201481116-b157270c-8649-42af-b9bb-47c780ca0c7d.PNG)